docs/docs/guides/deploy/linux.mdx

```diff
@@ -19,8 +19,15 @@ cockroach start-single-node --insecure --background --http-addr :9090 --listen-a
 
 ## Install ZITADEL
 
+Download the ZITADEL release according to your architecture from [Github](https://github.com/zitadel/zitadel/releases/latest) 
+
+## Unpack the archive
+move to your download location and unpack the archive
+
 ```bash
-curl -s https://api.github.com/repos/zitadel/zitadel/releases/tags/v2.0.0 | grep "browser_download_url.*zitadel_Linux_$(uname -m).tar.gz" | cut -d '"' -f 4 | xargs wget -qO - | sudo tar --extract --gzip --overwrite --directory /usr/local/bin zitadel && sudo chown $(id -u):$(id -g) /usr/local/bin/zitadel
+#unpack and copy to /usr/local/bin
+LATEST=$(curl -i https://github.com/zitadel/zitadel/releases/latest | grep location: | cut -d '/' -f 8 | tr -d '\r'); wget -qO-  https://github.com/zitadel/zitadel/releases/download/$LATEST/zitadel_Linux_$(uname -m).tar.gz  | tar -xJ zitadel && sudo mv zitadel /usr/local/bin
+
```

Comment:
This command is not working on an ubuntu 😁

Reply:
![image](https://user-images.githubusercontent.com/9879976/183398145-59931c01-3954-4d9a-b8dd-5a57f1812690.png)

Reply:
thanks, I will test and fix this in an ubuntu container

Reply:
fixed